metadata-ingestion/src/datahub/ingestion/source/kafka_connect/sink_connectors.py

```diff
@@ -175,7 +175,7 @@ class BigQuerySinkConnector(BaseConnector):
     class BQParser:
         project: str
         target_platform: str
-        sanitizeTopics: str
+        sanitizeTopics: bool
```

Comment:
was the type annotation here previously wrong?

Reply:
Yes